### ISSUE TITLE
events calendar view not rendering on staging

### ISSUE THREAD

**Issue by alee** [comses/core.comses.net#178]
```
Uncaught TypeError: i is not a function
    at Object.<anonymous> (event_calendar.38c49fd7cc3d167874f3.js:1)
    at __webpack_require__ (bootstrap 1c0198596befca6e5809:54)
    at Object.<anonymous> (event_calendar.38c49fd7cc3d167874f3.js:1)
    at __webpack_require__ (bootstrap 1c0198596befca6e5809:54)
    at webpackJsonpCallback (bootstrap 1c0198596befca6e5809:25)
    at event_calendar.38c49fd7cc3d167874f3.js:1
```

**Comment by cpritcha** [comment]
Should be fixed by 03a481ae096cea597e23219833ce7c11e64337be

**Comment by alee** [comment]
Thanks!
